bin/p2-inspect/README.md

```diff
@@ -0,0 +1,67 @@
+# p2-inspect
+
+`p2-inspect` is a tool for examining the state of a p2 cluster. Given the address of any consul agent, it will pull the manifest SHAs and health checks for all the pods running in the cluster, across all its nodes. Example:
+
+```bash
+$ p2-inspect | python -mjson.tool
+```
+
+```json
+{
+    "isup": {
+        "aws1.example.com": {
+            "health_check": {
+                "output": "[/data/pods/isup/isup/installs/isup_vsjlzlxvnkizuxmkutqmkqhwukyryztuxhusnkpm/bin/launch]\n[PATH=/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin:/bin TERM=linux RUNLEVEL=3 PREVLEVEL=N UPSTART_EVENTS=runlevel UPSTART_JOB=runit UPSTART_INSTANCE= CONFIG_PATH=/data/pods/isup/config/isup_717cc0d58df240e2668c865cdc063d715446e6db.yaml]\n",
```

Comment:
Yeah it echoes `$@` and `env` in the HTTP response. I mainly wanted to confirm (when I was implementing the health checks last week) that the response was getting captured by consul. Seeing it in the output of `p2-inspect` might be useful for debugging.